pass-bakery/conf/routes

```diff
@@ -8,3 +8,6 @@ GET     /                           controllers.HomeController.index()
 
 # Map static resources from the /public folder to the /assets URL path
 GET     /assets/*file               controllers.Assets.versioned(path="/public", file: Asset)
+
+# API endpoints
+GET /pass-bakery/status             controllers.APIController.echo()
```

Comment:
Make sure to have the function name match its purpose. `echo()` was a good start, but its not really `echo()`, but rather `status()` or `serviceStatus()` what would be more readable and descriptive of purpose